Cabal/tests/ParserTests/regressions/nothing-unicode.golden

```diff
@@ -0,0 +1,24 @@
+name: 無
+version: 0
+cabal-version: >=1.10
+build-type: Simple
+license: UnspecifiedLicense
+synopsis: The canonical non-package 無
+x-無: 無
```

Comment:
ping @hvr 

It seems that ReadP parser accepts this. I'm 99% sure that all field-names on Hackage are ASCII  though. Should I add a `cabal chech`. Also what would be proper regexp for fieldnames. Currently it's *almost everything*:

```
$namecore        = [$printable] # [$space \: \" \{ \} $paren $symbol']
```

as field names are taken as is, but they can be validated further in the parser stage.